### PR TITLE
Update gevent + indent ?

### PR DESCRIPTION
**Pull request by pthegner**
# Description

This PR try to fix the errors of Loop Exit and set in rmq

## Issue

Issue link: BOT-1616

### CODE REVIEW

**Comment by pthegner** [comment]
retest this please

File: chaos/navitia.py [hunk 1]
```diff
@@ -112,17 +112,17 @@ def get_pt_object(self, uri, object_type, pt_objects=None):
 
         if response:
             json = response.json()
+            timeout = app.config['CACHE_CONFIGURATION'].get('NAVITIA_CACHE_TIMEOUT', 3600)
```
Comment (dvdn) [inline chaos/navitia.py]:
```suggestion
            cache_timeout = app.config['CACHE_CONFIGURATION'].get('NAVITIA_CACHE_TIMEOUT', 3600)
```

as already used for Navitia timeout

File: chaos/navitia.py [hunk 2]
```diff
@@ -112,17 +112,17 @@ def get_pt_object(self, uri, object_type, pt_objects=None):
...
+
             if pt_objects and json[pt_objects]:
                 json_pt_objects = json[pt_objects]
-                cache.set(cache_key, json_pt_objects,
-                          app.config['CACHE_CONFIGURATION'].get('NAVITIA_CACHE_TIMEOUT', 3600))
+                cache.set(cache_key, json_pt_objects, timeout)
```
Comment (dvdn) [inline chaos/navitia.py]:
```suggestion
                cache.set(cache_key, json_pt_objects, cache_timeout)
```

File: chaos/navitia.py [hunk 3]
```diff
@@ -112,17 +112,17 @@ def get_pt_object(self, uri, object_type, pt_objects=None):
...
                 return json_pt_objects
 
             if self.collections[object_type] in json and json[self.collections[object_type]]:
                 json_pt_object = json[self.collections[object_type]][0]
                 try:
-                    cache.set(cache_key, json_pt_object,
-                            app.config['CACHE_CONFIGURATION'].get('NAVITIA_CACHE_TIMEOUT', 3600))
+                    cache.set(cache_key, json_pt_object, timeout)
```
Comment (dvdn) [inline chaos/navitia.py]:
```suggestion
                    cache.set(cache_key, json_pt_object, cache_timeout)
```

**Pull request merged**